frontend/src/model/beta/releases/ReleaseForm.tsx

```diff
@@ -63,93 +69,112 @@ export default function ReleaseForm({
     onMinorReleaseChange(checked)
   }
 
+  useEffect(() => {
```

Comment:
You could replace the `useState` and `useEffect` with a single `useMemo` like so:
```
const latestRelease = useMemo(() => releases && releases.length > 0 ? releases[0].semver : '', [])
```